src/utils/versionedTx.ts

```diff
@@ -11,7 +13,41 @@ import {
 import connection from 'state/connection/connection'
 import { filterTruthy } from 'utils'
 
-const lookupTablesByPool = import(/* webpackChunkName: 'lookupTablesByPool' */ 'constants/lookupTablesByPool')
+const lookupTablesByPool = (async () => {
```

Comment:
add try catch

Reply:
Try/catch is not neccessary/not enough since this run in an async thread, it wont crash so we dont need to catch it.

If we need to improve it, we might add the hold retry mechanism. I will do it later :D